src/pages/CreateEventPage/CreateEventPage.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react';
+import { CreateEventWrapper, Form } from './styled';
+import { parseISO, formatISO, format } from 'date-fns';
+import createEvent from './components/createEvent';
+import isFormFilled from './components/isFormFilled';
+import { InputField, TextArea, Select } from './components/FormFields';
+
+function CreateEvent() {
+  const [name, setName] = useState('');
+  const [image, setImage] = useState('');
+  const [imageAlt, setImageAlt] = useState('');
+  const [time, setTime] = useState(formatISO(Date.now()));
+  const [description, setDescription] = useState('');
+  const [eventType, setEventType] = useState('Choose event type');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  let createWasSuccess = false;
+  const eventOptions: string[] = [
+    "Subject matter event",
+    "Coding event",
+    "Project presentation",
+    "Workshop"
+  ]
+
+  function submitEvent() {
+    createEvent(time, name, description, firstName, lastName, email)
+      .then((result) => {
+        createWasSuccess = true;
+      })
+      .catch(() => {
+        console.log('Create failed');
+      });
+  }
+
+  return (
+    <CreateEventWrapper>
+      <h3>Create new event</h3>
+      <Form>
+        <InputField
+          id="name"
+          name="name"
+          label="Event name"
+          type="text"
```

Comment:
Instead of having a generic `InputField` component, you should probably define a `TextField`, `DateField` and `TimeField` component. Then you'd avoid having to repeat the type every time and we'd get greater flexibility if we want to change the `DateField` and `TimeField` later - e.g. maybe we want to use a custom widget for selecting date and time.